### DISCUSSION TITLE
mssql server

### DISCUSSION THREAD

**Discussion by Abdallah-Tah** [damms005/devdb-vscode#38] [General]
Is there a way to support the mssql server?

**Comment by damms005** [comment]
It is not in my immediate plan to add this. This is mostly because I do not use mssql.

However, it should be really easy to add this, because I imagine it will be is similar to the existing MySQL support.

Basically, there are 3 steps involved:
1. Create a mssql engine. See [current MySQL implementation](https://github.com/damms005/devdb-vscode/blob/main/src/database-engines/mysql-engine.ts) for example
2. Create [a provider](https://github.com/damms005/devdb-vscode/tree/main/src/providers) for mssql
3. Allow it loadable in the config file. See [current MySQL implementation](https://github.com/damms005/devdb-vscode/blob/main/src/providers/config-file-provider.ts#L36) for example

Then PR your changes. I can then review and help where necessary.

Thanks for checking this out!

**Reply by damms005** [reply]
This is now added.

Upgrade to latest version and enjoy!

**Comment by PratikIIH** [comment]
i found that when new column add or remove from mySQL database table, the refresh button in devDb won't able to update that migration changes , we have to close and than reconnect to database. please check this

**Reply by damms005** [reply]
Please [open an issue](https://github.com/damms005/devdb-vscode/issues/new?assignees=&labels=bug%2Cunconfirmed%2Clow+priority&projects=&template=bug_report.yml) for this.

**Reply by damms005** [reply]
@PratikIIH This is now fixed in the latest version. Update and enjoy! 🍸
